vignettes/special-files.Rmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "Special files"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Special files}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+#| include: false
+library(testthat)
+```
+
+This vignette describes the various special files that testthat works with: test, helper, and setup files.
+
+## Test files
+
+These are bread and butter of testthat.
+Test files live in `tests/testthat`, start with either `test-` or `test_`, and end with `.r` or `.R`.
+We recommend organising your test files so that there's a one-to-one correspondence between the files in `R/` and the files in `tests/testthat/` so that (e.g.) `R/file-one.R` has a matching `tests/testthat/test-one.R`.
```

Comment:
```suggestion
We recommend organising your test files so that there's a one-to-one correspondence between the files in `R/` and the files in `tests/testthat/` so that (e.g.) `R/file-one.R` has a matching `tests/testthat/test-file-one.R`.
```

Bringing this into line with what `usethis::use_test()` would actually do.

Reply:
Do you want to mention `usethis::rename_files()` somewhere around here? OTOH I suppose `use_r()` / `use_test()` talk about the process for existing packages (including `rename_files()`), so you could just assume people will learn about it there.